README.md

```diff
@@ -571,6 +573,46 @@ We have a small collection of algorithms, one for every day of the month. Scroll
       you       2
       ```
 
+## **December 27 - Cup and String Phone**
+  - **Problem :**
+    - Tharun is playing with his friends using a string phone. To make this interesting he asked his friends to join him. He asked them to bring cup of their own and he will bring the strings for all of them so that all can communicate with each other. Now he wants to find the number of strings required to communicate with his friends. Help him with a program that takes a number of friends as input and give the number of strings as output.
```

Comment:
hyperlink string phone to something like https://www.google.com/search?q=string+phone&source=lnms&tbm=isch